Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityArticulatedHand.cs

```diff
@@ -5,17 +5,23 @@
 using Microsoft.MixedReality.Toolkit.Input;
 
 #if UNITY_WSA
-using UnityEngine;
 using UnityEngine.XR.WSA.Input;
-#endif
-
-#if WINDOWS_UWP
-using Microsoft.MixedReality.Toolkit.Windows.Utilities;
+#if WINDOWS_UWP || DOTNETWINRT_PRESENT
```

Comment:
later in the file, the DOTNETWINRT_PRESENT define is paired with UNITY_WSA... is there intended to be different conditions where this define is checked?

Reply:
Sometimes the checks are already within a larger `#if UNITY_WSA` check (like this case, a few lines higher), so I chose not to duplicate it.

Reply:
i see. sounds good.